test/test_databases/test_database.py

```diff
@@ -8,7 +8,9 @@
 
 @pytest.fixture
 def dcs():
-    return DatabaseCreateStatement()
+    db = DatabaseCreateStatement()
+    db.DO_PARSE_BOOLS = True
```

Comment:
I think it would be helpful to have a separate set of test cases for when this config flag is set to False (which is the default setting for it). So that we can confirm that when this is set to False, we're not parsing '0' and '1' values as Booleans. Same goes for the mysql, postgres and redshift tests.